cpp/src/cider/tests/utils/CiderArrowChecker.cpp

```diff
@@ -69,6 +69,52 @@ bool checkArrowBuffer(const struct ArrowArray* expect_array,
   return true;
 }
 
+template <typename T>
+bool checkArrowBufferDecimal(const struct ArrowArray* expect_array,
+                             const struct ArrowArray* actual_array) {
```

Comment:
keep value check in this func, encapsulate and reuse arrow buffer check part in `checkArrowBuffer`?